tests/e2e/default/sms/send-message.wdio-spec.js

```diff
@@ -60,10 +61,10 @@ describe('Send message', () => {
     await messagesPage.sendMessage(
       smsMsg(healthCenter.name),
       healthCenter.name,
-      `${healthCenter.name} - all  contacts`
+      `${healthCenter.name} - all`
```

Comment:
There's a weird thing happening here. The translation has an extra parameter that is empty, so when rendering there are theoretically 2 spaces between `all` and `contacts`, but that additional space makes selecting the element through text impossible.